doc/build/core/constraints.rst

```diff
@@ -697,20 +703,22 @@ normally by using a convention which includes ``%(constraint_name)s``
 and then applying a name to the type::
 
     metadata = MetaData(
-        naming_convention={"ck": "ck_%(table_name)s_%(constraint_name)s"}
+        naming_convention={"type_ck": "type_ck_%(table_name)s_%(constraint_name)s"}
```

Comment:
I think I might want to keep the rendered example prefix as "ck_", because, when database identifiers get too long, things start getting weird.   at this point we have good truncation logic for these names but especially on a backend like Oracle names are truncated very short.

Reply:
ok.